test/unit/src/common/actions/t_repositories.js

```diff
@@ -108,23 +110,33 @@ describe('repositories actions', () => {
         expect(requestActions.length).toBe(3);
       });
     });
-
   });
 
-  xcontext('searchRepos', () => {
+  describe('searchRepos', () => {
+    let store;
+
+    beforeEach(() => {
+      store = mockStore({});
+    });
 
     it('should create an action to update search term', () => {
-      // const expectedAction = {
-      //   type: ActionTypes.REPOSITORIES_SEARCH,
-      //   payload: 'test'
-      // };
-      //
-      // store.dispatch(searchRepos('test'));
-      // expect(store.getActions()).toInclude(expectedAction);
+      const expectedAction = {
+        type: ActionTypes.REPOSITORIES_SEARCH,
+        payload: 'test'
+      };
+
+      store.dispatch(searchRepos('test'));
+      expect(store.getActions()).toInclude(expectedAction);
     });
 
     it('should create a valid flux standard action', () => {
-      // expect(isFSA(action)).toBe(true);
+      const action = {
+        type: ActionTypes.REPOSITORIES_SEARCH,
+        payload: 'test'
+      };
+
+      store.dispatch(searchRepos('test'));
+      expect(isFSA(action)).toBe(true);
```

Comment:
We don't need to `dispatch` anything here (we are not even checking anything from the store).

We want to test if what `searchRepos` return is in Flux Standard Action format.

So: `expect(isFSA(searchRepos('test'))).toBe(true);` should be enough for whole test.